pmd-swift/src/main/java/net/sourceforge/pmd/lang/swift/ast/SwiftRootNode.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * BSD-style license; for more info see http://pmd.sourceforge.net/license.html
+ */
+
+package net.sourceforge.pmd.lang.swift.ast;
+
+import org.antlr.v4.runtime.ParserRuleContext;
+
+import net.sourceforge.pmd.lang.ast.AstInfo;
+import net.sourceforge.pmd.lang.ast.Parser.ParserTask;
+import net.sourceforge.pmd.lang.ast.RootNode;
+import net.sourceforge.pmd.lang.swift.ast.SwiftParser.SwTopLevel;
+
+// package private base class
+abstract class SwiftRootNode extends SwiftInnerNode implements RootNode {
```

Comment:
I was wondering, how/when this new class is actually used. But figured out -> antlr4-wrapper.xml